CMakeLists.txt

```diff
@@ -0,0 +1,1042 @@
+
+	if(${loop} STREQUAL sixvefox OR ${loop} STREQUAL sixscfox OR ${loop} STREQUAL sixda)
+		message(STATUS "Will run dafor on ${loop}.f")
+		add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${loop}.f COMMAND ${CMAKE_CURRENT_BINARY_DIR}/dafor ${loop}n.f ${loop}.f > ${loop}_dafor.log DEPENDS run_astuce)
+	else()
+		add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${loop}.f COMMAND ${CMAKE_COMMAND} -E copy ${loop}n.f ${loop}.f DEPENDS run_astuce)
+	endif()
+endforeach()
+
+###################################################################################################
+#linking
+###################################################################################################
+
+if(CERNLIB)
+#	if(EXISTS "/cern/pro/bin/cernlib")
+		#running on lxplus and friends
+#		SET_TARGET_PROPERTIES(SixTrack PROPERTIES LINK_FLAGS "`cernlib -u graflib packlib_noshift kernlib_noshift` -lg2c -lpthread")
+#	else()
+		add_library(g2c STATIC IMPORTED)
+		add_library(graflib STATIC IMPORTED)
+		add_library(grafX11 STATIC IMPORTED)
+		add_library(kernlib_noshift STATIC IMPORTED)
+		add_library(packlib_noshift STATIC IMPORTED)
+		add_library(X11 STATIC IMPORTED)
+		add_library(Xau STATIC IMPORTED)
+		add_library(xcb STATIC IMPORTED)
+		add_library(Xdmcp STATIC IMPORTED)
+
+		set_target_properties(g2c PROPERTIES IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/SixTrack/lib32/libg2c.a)
+		set_target_properties(graflib PROPERTIES IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/SixTrack/lib32/libgraflib.a)
+		set_target_properties(grafX11 PROPERTIES IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/SixTrack/lib32/libgrafX11.a)
+		set_target_properties(kernlib_noshift PROPERTIES IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/SixTrack/lib32/libkernlib_noshift.a)
+		set_target_properties(packlib_noshift PROPERTIES IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/SixTrack/lib32/libpacklib_noshift.a)
+		set_target_properties(X11 PROPERTIES IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/SixTrack/lib32/libX11.a)
+		set_target_properties(Xau PROPERTIES IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/SixTrack/lib32/libXau.a)
+		set_target_properties(xcb PROPERTIES IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/SixTrack/lib32/libxcb.a)
+		set_target_properties(Xdmcp PROPERTIES IMPORTED_LOCATION ${CMAKE_SOURCE_DIR}/SixTrack/lib32/libXdmcp.a)
+
+		find_library(pthread STATIC pthread)
+		find_library(dl STATIC dl)
+		find_library(gcc_eh STATIC gcc_eh)
+
+		target_link_libraries(SixTrack graflib grafX11 packlib_noshift kernlib_noshift g2c X11 xcb Xau Xdmcp dl pthread gcc_eh)
+#	endif()
+endif(CERNLIB)
+
+#if we are using crlibm, link to the crlibm library
+if(CRLIBM)
+	target_link_libraries(SixTrack crlibm)
+endif(CRLIBM)
+
+#if we are using merlinscatter, link to the crlibm library
+if(MERLINSCATTER)
+	target_link_libraries(SixTrack merlinscatter)
+endif(MERLINSCATTER)
+
+#if we are using root, link to all the root libraries
+if(ROOT)
```

Comment:
I thought we discussed to kill the CERNLIB (PAW?) stuff and replace it with root bits?